src/api/spec/controllers/webui/configuration_controller_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ConfigurationController do
+  let(:confirmed_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+
+  describe 'GET #index' do
```

Comment:
Yep, will do that with a shoulda matcher and test the before filter in the webui controller test!